homeassistant/components/totalconnect/__init__.py

```diff
@@ -1 +1,51 @@
 """The totalconnect component."""
+import logging
+import voluptuous as vol
```

Comment:
Add a blank line between standard library, 3rd party and homeassistant import groups.

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
done

Reply:
Please add a blank line between logging and voluptuous.

Reply:
done